server/lib/remote-address.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Utility function to parse the client IP address from request headers.
+
+'use strict';
+
+const CLIENT_IP_ADDRESS_DEPTH = require('./configuration').get('clientAddressDepth') || 1;
```

Comment:
Just because that's what the old code did:

https://github.com/mozilla/fxa-content-server/blob/v1.98.1/server/lib/logging/route_logging.js#L28

No idea what the original justification for that was, but I don't want to risk breaking anything by changing it.